### PR TITLE
Proposal for creating VMware SIG

### PR DESCRIPTION
**Pull request by Akasurde**
Signed-off-by: Abhijeet Kasurde <akasurde@redhat.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,6 +24,7 @@ This repository serves as a space for the Ansible Community Working Group to man
   - [Network Working Group](https://github.com/ansible/community/tree/master/group-network)
   - [Testing Working Group](https://github.com/ansible/community/tree/master/group-testing)
   - [Windows Working Group](https://github.com/ansible/community/tree/master/group-windows)
+  - [VMware Working Group](https://github.com/ansible/community/tree/master/group-vmware)
```
Comment (dagwieers) [inline README.md]: Sort alphabetically :-)

Comment (Akasurde) [inline README.md]: OK

Comment (Akasurde) [inline README.md]: Done.

**Comment by Akasurde** [comment]
@dagwieers Thanks for review and comments.

**Pull request merged**